src/test/util_tests.cpp

```diff
@@ -145,9 +145,17 @@ BOOST_AUTO_TEST_CASE(util_Join)
     BOOST_CHECK_EQUAL(Join<std::string>({"foo", "bar"}, ", ", op_upper), "FOO, BAR");
 }
 
-BOOST_AUTO_TEST_CASE(util_FormatISO8601DateTime)
+BOOST_AUTO_TEST_CASE(util_FormatParseISO8601DateTime)
```

Comment:
What is `FormatParseISO8601DateTime`? I suggest add a separate test case for `ParseISO8601DateTime`.

Reply:
Well, if I'll do so at which one of them should I add the round trip test? :)

Reply:
To the new case IMO.

Reply:
I think testing both in one test case is fine in this case. It's not like it's a huge function that needs to be broken up.

Reply:
Yeah it's fine, just that we name cases like "util_<function>".